### DISCUSSION TITLE
ScriptReference/Mathf.DeltaAngle

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#198] [Notes]
# ScriptReference/Mathf.DeltaAngle

https://docs.unity3d.com/ScriptReference/Mathf.DeltaAngle.html

**Comment by mvi** [comment]
Note this value is signed and can produce negative values, for example  
`Debug.Log(Mathf.DeltaAngle(1080, 90));`  
will print `90`  
`Debug.Log(Mathf.DeltaAngle(90, 1080));`  
will print `-90`
